docs/applications/repl.md

```diff
@@ -0,0 +1,76 @@
+# REPL
+
+`REPL` is a minimal application based on the JupyterLab code console.
+
+![image](https://user-images.githubusercontent.com/591645/153935929-23a5d380-363e-490b-aabd-f0a780140588.png)
+
+## Embedding the REPL on another website
+
+You can embed the REPL on any website with the following code snippet:
+
+```html
+<iframe
+  src="https://jupyterlite.rtfd.io/en/latest/_static/lab/index.html"
```

Comment:
all point at `lab` instead of `repl`...

Reply:
also somewhat uneasy about repping the docs URL directly as a thing to be used hot... perhaps an activated version of the docs, once available would be less concerning...

Reply:
Good point, we can mention folks should use a custom deployment if possible, and maybe we can point to `jupyterlite.rtfd.io/repl` or `jupyterlite.github.io/demo/repl` (a bit more stable) as an example.

Reply:
Updated in https://github.com/jupyterlite/jupyterlite/pull/498/commits/9aaeca6bdb675465e9a9b94c40795f52bc919292